### ISSUE TITLE
[Feature request]: Separate the installation of flatpaks from traditional packages

### ISSUE THREAD

**Issue by Julynx** [flatpak/flatpak#4988]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a feature request that matches the one I want to file, without success.

### Suggestion

I love flatpak and how it automatically manages dependencies in a distribution agnostic way. I also like how it allows for new ways of permission control. 

However, as it currently stands I believe it doesn't communicate properly what it is and what it can do to the user.

Right now, flatpak still doesn't have its own official graphical client, flatpaks are usually integrated into each distribution's "store" instead.

This causes problems for new users that you will probably have suffered yourself. For example:

- "I don't know if I have the flatpak or the .deb installed"
- "Why are there two 'versions' listed for this app (.deb and flatpak) and which one should I install?"
- "I already had X app installed through flatpak and I forgot and installed the .deb and now I have two different versions of the same app lmao"
- etc.

But the most fundamental problem is that distributions are still having to dedicate effort to integrate flatpaks side by side with their own packages in their store and flatpak is still regarded as another package manager and used in parallel with apt or dnf. This doesn't reflect what flatpak is to the user or what it can and should be.

A valuable lesson that flatpaks have taught us is that users, distributions, and software developers shouldn't have to think about how userspace apps are packaged and distributed. For this, userspace apps pull dependencies and update "on their own" and do their work without interacting too much with system components. 

This shows that Flatpaks exist in a layer closer to the user and separated from distribution-managed packages. Serving packages inside an independent store would better reflect this separation of purpose and responsibility to the user.

This is why there should be a "flatpak store" that works as an independent app, ideally a flatpak, that looks and runs the same in each and every linux distribution (just like flatpaks do) and should only contain flatpak versions of apps because that is the only way userspace apps should be fundamentally managed.

**Comment by chrisawi** [comment]
App stores should combine the Flatpak and native package versions of apps into a single entry, which is what GNOME Software does, at least where the app IDs match.

A Flatpak-only app store distributed via Flatpak has been [attempted before](https://gitlab.gnome.org/haecker-felix/souk), but regardless, it's out of scope for Flatpak itself.
